subprojects/wrapper/src/integTest/groovy/org/gradle/integtests/WrapperGenerationIntegrationTest.groovy

```diff
@@ -72,7 +72,7 @@ class WrapperGenerationIntegrationTest extends AbstractIntegrationSpec {
 
         then: "the checksum should be constant (unless there are code changes)"
         // TODO If you need to update this because of necessary code change in the wrapper, also address the TODO in 'wrapper.gradle.kts'
-        sha256(file("first/gradle/wrapper/gradle-wrapper.jar")).asHexString() == "e996d452d2645e70c01c11143ca2d3742734a28da2bf61f25c82bdc288c9e637"
+        sha256(file("first/gradle/wrapper/gradle-wrapper.jar")).asHexString() == "810cd5fb51fa916cd09c9332f5f60bdfd786871adb90e29a2b65cd74cf9730eb"
```

Comment:
BTW, I did not get what the `also address the TODO in 'wrapper.gradle.kts'` comment refers to, as I did not find a `wrapper.gradle.kts` file in the code base. I only found `gradlebuild.wrapper.gradle.kts`, but that file does not contain a `TODO`.

Reply:
Yeah, I don't know where this is supposed to be either. @jjohannes?

Reply:
Sorry the file was renamed. This is about `subprojects/wrapper/build.gradle.kts`.

https://github.com/gradle/gradle/blob/524b568298db23f84216840fa10a6f0177ce5248/subprojects/wrapper/build.gradle.kts#L50-L64

Reply:
I'm not quite sure I understand what the goal here is. To filter out "gradle-cli-classpath.properties" and "gradle-cli-parameter-names.properties" in the `executableJar` task so they do not get packaged to "gradle-wrapper.jar"? If so, what about the also packaged "gradle-wrapper-classpath.properties" and "gradle-wrapper-parameter-names.properties" files, should they stay?

Reply:
You can just remove the whole "TODO" block I linked above. The goal of it was to not change the Jar. These "properties" are currently in the Jar by mistake (caused by a broken setup that is now fixed). They are not needed at all.

Now, as the Jar changes anyway, we can remove the whole workaround. The only thing you need to address in code is the compiler warning you get in `org.gradle.wrapper.Install` after removing the block.

Reply:
Ok, done.

Reply:
Thanks @sschuberth! Can you also adjusts the `sha256` (see the line this comment thread is attached to) in the test again as it has now changed again due to the additional changes. Right now this test is failing.

Reply:
You will het the right sha by running the test and looking at the test failure.

Reply:
Yes, sure will do that later. Sorry for forgetting that.

Reply:
Done.